### PR TITLE
#163620272 Documented all endpoints on Swagger

### PR DESCRIPTION
**Pull request by ebzeal**
#### What does this PR do?
 - This PR documents all the endpoints of our REST API

#### How should this be manually tested?
 - clone repo 
 - switch to this branch 
 - Launch on local
 - view docs on localhost URL/docs
 -  When merged Docs should be up live on https://neon-ah-staging.herokuapp.com/api/v1/docs/#/

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2229851/stories/163620272

### CODE REVIEW

**Comment by rafmme** [comment]
LGTM, yet

**Comment by Makwe-O** [comment]
LGTM, yet

**Comment by Sammuel09** [comment]
LGTM, yet.

File: swagger.yaml [hunk 1]
```diff
@@ -6,373 +6,1408 @@ info:
...
+                  format: password
+                confirmPassword:
+                  type: string
+                  format: password
       responses:
-        '200':
+        '201':
           $ref: '#/components/responses/LoginSuccess'
+        '409':
+          $ref: '#/components/responses/ErrorBadRequest'
```
Comment (Sammuel09) [inline swagger.yaml]: Nice work, Shola. I think the status code should 400 for a bad request. Kindly fix this.

**Pull request merged**